Jenkinsfile

```diff
@@ -424,6 +425,26 @@ pipeline {
       }
     }
 
+    stage('L2: Intent and Slot Classification') {
+      when {
```

Comment:
What does it do? the when and anyOf?

Reply:
It is not clear. I had to resolve merge conflict in this file manually. Jenkins code seems to be fine, but in the test it run Green only until the middle and then stopped for some unclear reason.